templates/.pr-preview.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Thanks for this! Unfortunately, this file really needs to be generated dynamically based on what the user chooses (ReSpec or BikeShed). 

So, can definitely use this as a data structure to start with, and have it written out dynamically.